vortexor/src/cli.rs

```diff
@@ -113,4 +136,14 @@ pub struct Cli {
     /// SIGUSR1 signal to the vortexor process will cause it to re-open the log file.
     #[arg(long="log", value_name = "FILE", value_parser = clap::value_parser!(String))]
     pub logfile: Option<String>,
+
+    /// The address of RPC server to which the vortexor will forward transaction
+    #[arg(long, value_parser = parse_http_url, value_name = "URL", action = ArgAction::Append)]
```

Comment:
i believe arguments of type Vec are automatically marked as Append

Reply:
right. I removed and verified it works. Thanks